addons/website/views/website_templates.xml

```diff
@@ -5,18 +5,16 @@
     <t t-set="show_dropdown" t-value="(submenu.is_mega_menu and submenu.is_visible) or submenu.child_id.filtered(lambda menu: menu.is_visible)"/>
     <li t-if="submenu.is_visible and not (submenu.child_id or submenu.is_mega_menu)" t-attf-class="#{item_class or ''}">
         <a t-att-href="submenu.clean_url()"
-            t-attf-class="#{link_class or ''} #{'active' if submenu.clean_url() and unslug_url(request.httprequest.path) == unslug_url(submenu.clean_url()) else ''}"
+            t-attf-class="#{link_class or ''} #{'active' if submenu.is_active() else ''}"
```

Comment:
Maybe take the opportunity to write the condition as `#{submenu.is_active() and 'active'}"` ? Same below (or / else unnecessary).

Reply:
Sure, I kept the minimal diff but I'll change